stacker_blueprints/dynamodb.py

```diff
@@ -7,23 +7,24 @@
     Output,
 )
 
-from troposphere.route53 import RecordSetType
 from . import util
 
 MAX_GSI_VALUE = 5
+MAX_LSI_VALUE = 5
 
 
 def prep_schemata(config):
+    schemata = []
     try:
-        schemata = []
-        for schema in config["KeySchema"]:
-            schemata.append(dynamodb2.KeySchema(**schema))
-
+        schemas = config["KeySchema"]
     except KeyError:
         raise KeyError(
             "The key schema is required for the creation of a DynamoDB table."
```

Comment:
I think @mhahn commented on this, but might as well use `KeySchema` here, rather than `key schema`, just to make it more obvious (not that it's that confusing).